persistence/postgres-es/src/view_repository.rs

```diff
@@ -100,13 +101,18 @@ where
         };
         let version = context.version + 1;
         let payload = serde_json::to_value(&view).map_err(PostgresAggregateError::from)?;
-        sqlx::query(sql.as_str())
+        let rows_affected = sqlx::query(sql.as_str())
             .bind(payload)
             .bind(version)
             .bind(context.view_instance_id)
+            .bind(context.version)
             .execute(&self.pool)
             .await
-            .map_err(PostgresAggregateError::from)?;
+            .map_err(PostgresAggregateError::from)?
+            .rows_affected();
+        if rows_affected < 1 {
+            return Err(PersistenceError::OptimisticLockError);
```

Comment:
I *think* this should be coverable by a stress-test.
Are optimistic lock errors for committing events currently covered?